docs/data-sources/vcpu_server.md

```diff
@@ -0,0 +1,129 @@
+---
+subcategory: "Compute Engine"
+layout: "ionoscloud"
+page_title: "IonosCloud : ionoscloud_vcpu_server"
+sidebar_current: "docs-ionoscloud-datasource-vcpu-server"
+description: |-
+  Get information on a Ionos Cloud VCPU Servers
+---
+
+# ionoscloud\_vcpu_server
+
+The **VCPU Server data source** can be used to search for and return existing VCPU servers. 
+If a single match is found, it will be returned. If your search results in multiple matches, an error will be returned.
+When this happens, please refine your search string so that it is specific enough to return only one result.
+
+## Example Usage
+
+### By ID
+```hcl
+data "ionoscloud_vcpu_server" "example" {
+   datacenter_id = <datacenter_id>
+   id			 = <server_id>
+}
+```
+
+### By Name
+```hcl
+data "ionoscloud_vcpu_server" "example" {
+   datacenter_id = <datacenter_id>
+   name			 = "VCPU Server Example"
```

Comment:
indentr

Reply:
@cristiGuranIonos I look in the file and I see that the indentation is correct, not sure why `git diff` is showing a bad identation but I will leave this as it is.

Reply:
@cristiGuranIonos done in fdb4aa9b46872788cb95fb6ede380875592cd30e